cdap-client/src/main/java/co/cask/cdap/client/MetricsClient.java

```diff
@@ -135,9 +138,7 @@ public MetricQueryResult query(List<String> metrics, List<String> groupBys, Map<
    */
   // TODO: take in query object shared by MetricsHandler
   public MetricQueryResult query(List<String> metrics, List<String> groupBys, Map<String, String> tags,
```

Comment:
can we swap position of `tags` and `groupBys`, simply because `tags` would be used more frequently?